### ISSUE TITLE
UnicodeEncodeError: 'ascii' codec can't encode character u'\uf003'

### ISSUE THREAD

**Issue by denegen** [jezdez/django-staticfiles#7]
While executing 'manage.py collectstatic' I recieved the following trace:

```
Copying '/home/erwin/.virtualenvs/django-base/lib/python2.6/site-packages/django_crystal_small/static/crystal/48x48/apps/hibernate.png'
Traceback (most recent call last):
  File "./manage.py", line 11, in <module>
    execute_manager(settings)
  File "/home/erwin/.virtualenvs/django-base/lib/python2.6/site-packages/django/core/management/__init__.py", line 438, in execute_manager
    utility.execute()
  File "/home/erwin/.virtualenvs/django-base/lib/python2.6/site-packages/django/core/management/__init__.py", line 379, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/erwin/.virtualenvs/django-base/lib/python2.6/site-packages/django/core/management/base.py", line 191, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/home/erwin/.virtualenvs/django-base/lib/python2.6/site-packages/django/core/management/base.py", line 220, in execute
    output = self.handle(*args, **options)
  File "/home/erwin/.virtualenvs/django-base/lib/python2.6/site-packages/django/core/management/base.py", line 351, in handle
    return self.handle_noargs(**options)
  File "/home/erwin/.virtualenvs/django-base/lib/python2.6/site-packages/staticfiles/management/commands/collectstatic.py", line 88, in handle_noargs
    self.copy_file(path, prefixed_path, storage, **options)
  File "/home/erwin/.virtualenvs/django-base/lib/python2.6/site-packages/staticfiles/management/commands/collectstatic.py", line 190, in copy_file
    self.log("Copying '%s'" % source_path, level=1)
  File "/home/erwin/.virtualenvs/django-base/lib/python2.6/site-packages/staticfiles/management/commands/collectstatic.py", line 107, in log
    self.stdout.write(msg)
UnicodeEncodeError: 'ascii' codec can't encode character u'\uf003' in position 130: ordinal not in range(128)
(django-base)erwin@wpl082:~/Projects/django_base$ git freeze
```

This probably happens while coping the file home&food.png, note the '&' char.
Using 'django-staticfiles' version 1.0b1 and 'django-crystal-small' version '2011.2.13'.

**Comment by jezdez** [comment]
Fixed handling files of non-ASCII characters in findstatic and collectstatic management commend. Closed by 297158d70111f7403629aaa75b0581bc697444a0.

**Comment by denegen** [comment]
Thanks!
